packages/core/src/proto/peer_exchange.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+message PeerInfo {
+  optional bytes ENR = 1;
+}
+
+message PeerExchangeQuery {
+  optional uint64 numPeers = 1; // number of peers requested
```

Comment:
The spec says `int`. I am guessing the `uint64` is coming from the nwaku implementation.

Would it make sense to use `uint32` instead @kaiserd? In JavaScript `uint64` means we use `Bigint` which should not really be necessary considering the functionality of this field.